src/Gui/TaskMeasure.cpp

```diff
@@ -228,7 +276,7 @@ void TaskMeasure::update() {
         else {
             // Create measure object
             setMeasureObject(
-                (Measure::MeasureBase*)doc->addObject(measureType->measureObject.c_str(), measureType->label.c_str())
+                (Measure::MeasureBase*)createObject(_mMeasureType->measureObject)
```

Comment:
You are not allowed to include stuff from extension modules to the core system. This is a serious design flaw!

Reply:
This is addressed in a separate PR #15190